index.html

```diff
@@ -4,11 +4,168 @@
     <title>Mockup Home</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="/styles/styles.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script src="https://use.fontawesome.com/ca6c48c124.js"></script>
   </head>
   <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+
+    <!-- HEADER SECTION -->
+    <section id="header">
+      <h1 id="page_heading"><a href="index.html">Home Page</a></h1>
+      <h3 id="heading_subtext">The page for the home.</h3>
+      <div id="divider_container">
+        <div id="header_divider" class="divider"></div>
+      </div>
+      <h3 id="features_caption">Take a look at this amazing <strong>home page</strong>!</h3>
+      <div id="features_img_container">
+        <img class="features_img" src="/assets/sunset.jpg"/>
+        <img class ="features_img" src="/assets/sunset.jpg"/>
+        <img class="features_img" src="/assets/sunset.jpg"/>
+      </div>
+    </section>
+
+    <section id="features">
+    <div id="features_button_container">
+        <form action="learn-more.html">
+        <button type="submit" id="features_button" ><i class="fas fa-file" style="color: rgb(228, 207, 211); margin-right: 5px;"></i>Tell me more</button>
+        </form>
+    </div>
+    <h2 id="features_caption"></h2>
+    </section>
+
+    <!-- ALL BLOG POST CONTENT -->
+    <section id="main">
+      <div id="banner_img_container">
+        <img id="banner_img" src="/assets/sunsetcrop.png" />
+        <h3 id="img_quote">Use this space for a great quote</h3>
+      </div>
+
+      <div id="posts_container">
+
+        <!-- LEFT SIDE POSTS BELOW -->
+        <div id="left_container">
+            <div id="totally_container">
+                <h2 id="totally_heading">I don't want to say <strong>it's the aliens</strong>...<br>But it's the aliens.</h2>
+                <div class="blog_img_container">
+                    <img class="blog_img" src="/assets/sunset.jpg">
+                </div>
+                <h3 class="totally"><strong>Totally</strong>...</h3>
+                <p class="totally_para">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.</p>
+
+                <form action="continue-reading.html">
+                <button type="submit" id="totally_button"><i class="fas fa-file" style="color: rgb(228, 207, 211); margin-right: 5px;"></i>Continue Reading</button>
+                </form>
+                <div id="totally_divider" class="divider"></div>

```

Comment:
This div should either be centered or stretch the length of the left container.

Reply:
Whoops, yeah thanks for catching that - I definitely missed it, it was intended to stretch the length of the container. I'll fix that!